sdk/ios/Configsum/Configsum/Configsum/Configsum.swift

```diff
@@ -5,7 +5,7 @@
 //  Created by Alexandru Savu on 2017-10-04.
 
 public typealias failureBlock = (Error) -> Void
-public typealias successBlock = (Bool) -> Void
+public typealias successBlock = () -> Void
```

Comment:
@mictab repercussions for this were not addressed

Reply:
@alexsavu Thanks, I'll have a look